### PR TITLE
First commit

### PR DESCRIPTION
**Pull request by 0xGabi**

### CODE REVIEW

**Comment by sohkai** [comment]
@0xGabi Could we gitignore the node modules? There are a lot of files 😅

**Comment by ajsantander** [comment]
Could we add some content to the main README file? It would be nice to get an idea of what this repo is all about.

**Comment by 0xGabi** [comment]
@sohkai @ajsantander Before merge, I wonder if we should update the packages names to something like `contract-helpers-<package>`?

**Comment by ajsantander** [comment]
@0xGabi I'd -1 that. Some of them would have really ugly names, like:
@aragon/contract-helpers-test-helpers :confused: 

Also, having we'd break backwards compatibility for people that might already be using the packages with those names. This should only be a change of where the packages are stored (repo) but not their name.

File: packages/test-helpers/utils.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+        }
+
+        throw new Error(`Could not find error ${property} in error mapping`)
+      },
+      set () {
+        throw new Error('Unexpected set to error mapping')
+      }
+    })
+  }
+}
```
Comment (sohkai) [inline packages/test-helpers/utils.js]: Let's add [`numbers`](https://github.com/aragon/aragon-court/blob/development/test/helpers/numbers.js) as a separate file.

File: packages/test-helpers/events.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const getEvents = ({ logs = [] }, event) => logs.filter(l => l.event === event)
+const getEventAt = (receipt, event, index = 0) => getEvents(receipt, event)[index]
+const getEventArgument = (receipt, event, arg, index = 0) => getEventAt(receipt, event, index).args[arg]
+const getNewProxyAddress = (receipt) => getEventArgument(receipt, 'NewAppProxy', 'proxy')
```
Comment (sohkai) [inline packages/test-helpers/events.js]: Let's add [`decodeEvents`](https://github.com/aragon/dao-templates/blob/master/shared/helpers/events.js#L5) here.

File: packages/test-helpers/utils.js (hunk 1 above)
Comment (sohkai) [inline packages/test-helpers/utils.js]: Let's add [`node`](https://github.com/aragon/dao-templates/blob/master/shared/helpers/node.js) as a separate file.

**Comment by sohkai** [comment]
Was on the fence about `assertRole` was going to be, but I can see app devs using those to sanity check their ACL permissions. The `apps` helper is not that useful.

**Pull request merged**